client/src/app/main.cpp

```diff
@@ -53,10 +57,40 @@ int main(int argc, char **argv)
     parser.addHelpOption();
     QCommandLineOption noOverlayOption("nooverlay", i18n("Do not display the overlay during initial data loading"));
     parser.addOption(noOverlayOption);
+    QCommandLineOption configKeyOption("config-key", i18n("Internal feature: Return the value for setting <key>"), "key");
+    parser.addOption(configKeyOption);
+    QCommandLineOption configValueOption("config-value", i18n("Internal feature: Set the value for setting specified by 'configKey' parameter"), "value");
```

Comment:
Ah, how about naming this option "set-config-value" ? It wasn't obvious when reading your example command line that this was a setter, not a getter.